compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/InterfaceLowering.kt

```diff
@@ -108,15 +115,18 @@ internal class InterfaceLowering(val context: JvmBackendContext) : IrElementTran
                 Visibilities.isPrivate(function.visibility)
```

Comment:
It's true for private non-JvmDefault, private JvmDefault should be keeped in interface

Reply:
Ah, but now we're evolving the feature, right? Private interface members require Java 9, AFAIK, so we should be guarding it with target JVM.

Reply:
JvmDefault requires target 8. Java 9 supports private interace members, but already JVM 8 allows to use private members in bytecode: https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.6